templates/default/tests/markdownFeatures.spec.js

```diff
@@ -0,0 +1,136 @@
+const { test, expect } = require('@playwright/test');
+const { MarkdownPage } = require('./markdown-page');
+
+test.describe('wiki links', () => {
```

Comment:
```suggestion
test.describe.parallel('wiki links', () => {
```

You can add `.parallel` to every test group so that the tests grouped within it don't run one after the other but in parallel. It will remove the warning that's currently shown in the console about the slow test files - and will basically speed up the process :)